src/form-host/index.js

```diff
@@ -11,12 +11,9 @@ window.loadForm = (formName, formType, formHtml, formModel, formXml, content, us
 };
 
 const loadForm = async (FormWireup, formName, formHtml, formModel, formXml, content, userSettingsDoc, contactSummary) => {
-  const wireup = new FormWireup(formHtml, formModel, formXml, userSettingsDoc, contactSummary);
-  const form = await wireup.render(content);
-  const formFiller = new FormFiller(form, { verbose: true });
-
-  window.form = form;
-  window.formFiller = formFiller;
```

Comment:
this actually matters to me a lot - i use `window.formFiller` in explorer to test and debug formFiller pretty frequently. it's an important dev scenario for me.  could we add this back - even if it is just in explorer?

Reply:
i don't care about `window.form`